services/metadata/conf/main/logback.xml

```diff
@@ -11,7 +11,7 @@
   </appender>
 
 
-  <Logger name="donkey" level="debug" />
+  <Logger name="terrain" level="debug" />
```

Comment:
Probably out of scope. We can change it later, if we want/it matters, presumably.